backend/src/main/java/com/benine/backend/http/CreatePresetHandler.java

```diff
@@ -86,7 +87,7 @@ public void handle(String s, Request request, HttpServletRequest req, HttpServle
    * @throws StreamNotAvailableException  If the camera does not have a stream.
    * @throws IOException  If the image cannot be written.
    */
-  private void createImage(int cameraID, int presetID) throws
+  private static void createImage(int cameraID, int presetID) throws
```

Comment:
We do have a factory for presets in the autopresetcration branch I think we should use that here. You can cherry pick the commits I think.
https://github.com/Kingdorian/Contextproject-BeNine/commit/204fa29b2aee924fd89d9efa071f9ded2006a782
https://github.com/Kingdorian/Contextproject-BeNine/commit/67b4e4d52549f70f1a67f2ad0fbe63bdebe7917e
https://github.com/Kingdorian/Contextproject-BeNine/commit/a3ebfeaf50460f052b0242bfbe164fed22c3b1b0